tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from ocs_ci.ocs import ocp
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import TimeoutSampler, get_latest_ocp_version
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.ocs.cluster import CephCluster, CephHealthMonitor
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: add image type validation (ga to ga , nightly to nightly, newer than current etc.)
+
+
+class TestUpgradeOCP(ManageTest):
+    """
+    1. check cluster health
+    2. check OCP version
+    3. perform OCP upgrade
+    4. check all OCP ClusterOperators
+    5. check OCP version
+    5. monitor cluster health
+    """
+
+    def test_upgrade_ocp(self):
+        """
+        Tests OCS stability when upgrading OCP
+
+        """
+
+        ceph_cluster = CephCluster()
+        with CephHealthMonitor(ceph_cluster):
+
+            ocp_upgrade_version = config.UPGRADE['ocp_upgrade_version']
```

Comment:
```suggestion
            ocp_upgrade_version = config.UPGRADE.get('ocp_upgrade_version')
```

Otherwise the logic here won't work. I pointed this .get few times here in other comments.

Reply:
fixed